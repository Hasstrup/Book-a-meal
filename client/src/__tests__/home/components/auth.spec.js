import React from 'react';
import renderer from 'react-test-renderer';
import AuthComponent from '../../../modules/home/components/auth'


describe('Auth component', () => {
  it('should match this snapshot', () => {
    const tree = renderer.create(<div> Hello</div>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
