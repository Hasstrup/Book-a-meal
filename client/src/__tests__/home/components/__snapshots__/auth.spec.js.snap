// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth component should match this snapshot 1`] = `
<div>
   Hello
</div>
`;
